nabs/visualizations.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from .plan_stubs import get_sample_info, snake_grid_list
+
+
+def show_shot_targets(sample_name, path, snake_like=True):
+    """
+    Display a plot with targets.
+
+    Displays a scatter plot with the targets that have been shot and the
+    targets that are still available.
+
+    Parameters
+    ----------
+    sample_name : str
+        The name of the sample file to plot the graph for.
+    path : str
+        The path of the sample file.
+    snake_like : bool
+        Indicates if the targets have been shot in a snake_like pattern.
+        # TODO this is a bad assumption - but most likely they will be
+        shot in a snake-like pattern....
+    """
```

Comment:
This docstring needs some reference to what it's used for- like the specific class it's designed for. Otherwise, coming across this while browsing the code is very confusing.

Reply:
Also, I'm starting to see what you meant when you said you thought all of this stuff should be bundled together... hmm

Reply:
I updated a bit the docstring for this function (97f39c1) - I will update it further if/when I change the `snake_like` pattern situation.